examples/logical_interconnect_groups.py

```diff
@@ -34,193 +34,311 @@
+                            {
+                                "relativeValue": 82,
+                                "type": "Port"
+                            },
+                            {
+                                "relativeValue": 3,
+                                "type": "Bay"
+                            },
+                            {
+                                "relativeValue": 1,
+                                "type": "Enclosure"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "desiredSpeed": "Auto",
+                    "logicalLocation": {
+                        "locationEntries": [
+                            {
+                                "relativeValue": 87,
+                                "type": "Port"
+                            },
+                            {
+                                "relativeValue": 3,
+                                "type": "Bay"
+                            },
+                            {
+                                "relativeValue": 1,
+                                "type": "Enclosure"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "desiredSpeed": "Auto",
+                    "logicalLocation": {
+                        "locationEntries": [
+                            {
+                                "relativeValue": 82,
+                                "type": "Port"
+                            },
+                            {
+                                "relativeValue": 6,
+                                "type": "Bay"
+                            },
+                            {
+                                "relativeValue": 2,
+                                "type": "Enclosure"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "desiredSpeed": "Auto",
+                    "logicalLocation": {
+                        "locationEntries": [
+                            {
+                                "relativeValue": 88,
+                                "type": "Port"
+                            },
+                            {
+                                "relativeValue": 6,
+                                "type": "Bay"
+                            },
+                            {
+                                "relativeValue": 2,
+                                "type": "Enclosure"
+                            }
+                        ]
+                    }
+                }
+            ],
+            "networkType": "Ethernet",
+            "ethernetNetworkType": "ImageStreamer",
+            "name": "deploy"
+        },
+        {
+            "networkUris": [
+                "/rest/ethernet-networks/26cb6480-23d0-4391-b481-ae524dd54f67"
```

Comment:
if possible try to make use of data parameter of ethernet network resource to set this URI

Reply:
done